tests/test_openjdk.py

```diff
@@ -280,3 +297,20 @@ def test_jdk_cassandra(container_per_test):
     container_per_test.connection.check_output(
         f"cd /tmp/{cassandra_base}/tools/bin/ && ./cassandra-stress write n=1 && ./cassandra-stress read n=1",
     )
+
+
+@pytest.mark.parametrize(
+    "container_per_test",
+    FIPS_OPENJDK_IMAGES,
+    indirect=True,
+)
+def test_openjdk_sec_providers(container_per_test: ContainerData) -> None:
+    """
+    Verifies that the primary security provider in FIPS-enabled OpenJDK
+    containers is `SunPKCS11-NSS-FIPS`. The test uses Java scripts to list and
+    validate security providers, ensuring FIPS compliance.
+    """
+    c = container_per_test.connection
+
+    c.run_expect([0], "java JCEProviderInfo.java")
```

Comment:
shorter and more in line with the rest of the code:
```suggestion
    c.check_output("java JCEProviderInfo.java")
```